third_party/com_github_mariusvanderwijden_fuzzyvm.patch

```diff
@@ -0,0 +1,159 @@
+diff -urN fuzzyvm-patch-a/generator/precompiles/BUILD.bazel fuzzyvm-patch-b/generator/precompiles/BUILD.bazel
+--- fuzzyvm-patch-a/generator/precompiles/BUILD.bazel	2025-11-19 15:04:38
++++ fuzzyvm-patch-b/generator/precompiles/BUILD.bazel	2025-11-19 15:05:22
```

Comment:
really not sure if this patch is needed ... i tried updating to latest but it still failed